fetchDependencies

```diff
@@ -304,7 +304,7 @@ if [ ! "$SPIRV_CROSS_ROOT" = "" ]; then
 	rm -rf ${REPO_NAME}
 	ln -sfn ${SPIRV_CROSS_ROOT} ${REPO_NAME}
 else
-	REPO_URL="https://github.com/KhronosGroup/${REPO_NAME}.git"
+	REPO_URL="https://github.com/js6i/${REPO_NAME}.git"
```

Comment:
Maybe return KhronosGroup?

Reply:
For the time being He needs to pull from his fork of SPIRV-CROSS a lot of GS work is done there too.